README.md

```diff
@@ -6,12 +6,32 @@
 
 The project follows the guideline of [the odin project](https://theodinproject.com/courses/javascript/lessons/todo-list?ref=lnav) to build a todo app using javascript for DOM manipulation and interaction. It uses javascript objects and functions. It uses webpack. It also uses Bootstrap for user interface design.
 
-It demonstrates the use of Object Oriented Programming in javascript.
+It demonstrates the use of Object Oriented Programming principle in javascript.
 
-For learning purpose:
+Features:
 
-- Some codes are WET
-- Some codes have been made DRY.
+- Add project
+- Add todo
+- Edit todo
+- Update todo status as completed or incomplete todo
+- Delete todo
+- Undo delete todo feature
```

Comment:
I think we can add that it restores the last deleted todo

Reply:
Thanks for the review. I'll add this.